toptal/picasso
# feat: add picasso-pictograms package

[FX-3636](https://toptal-core.atlassian.net/browse/FX-3636)

### Description

This pull request adds `@toptal/picasso-pictograms` package along with two initial pictograms.

Plan to split work on [FX-3636](https://toptal-core.atlassian.net/browse/FX-3636):

- first pull request adds the `@toptal/picasso-pictograms` package and two pictograms;
- second pull request adds [all the remaining pictograms from Figma](https://www.figma.com/file/6UPpJrQbP02OqBjtGL6KTr/Product-Pictograms?node-id=20%3A5168&t=qjUeLdbWgGJEFQVC-0).

### How to test

- Check Storybook temploy
- Check out updated `README` file (https://github.com/toptal/picasso/tree/fx-3636-illustration-package-for-picasso)
- Check `yarn generate:components-from-svg` command locally, there should be no diffs for already existing SVG React components
- All checks should pass

### Screenshots

https://user-images.githubusercontent.com/1390758/221236671-b8636325-c085-4fe8-b5ab-eece738e232d.mp4

### Development checks

- [x] Remove the SVGO workaround, research problem
- [N/A] Annotate all `props` in component with documentation
- [N/A] Create `examples` for component

<img width="775" alt="Screenshot 2023-02-27 at 12 00 14" src="https://user-images.githubusercontent.com/1390758/221519179-1c5b46b4-73f4-4191-9672-136c365f494a.png">

- [N/A] Covered with tests

**Breaking change**

- [N/A] codemod is created and showcased in the changeset
- [N/A] test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-3636]: https://toptal-core.atlassian.net/browse/FX-3636?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
